SUPPORT.md

```diff
@@ -30,7 +30,8 @@ Template variables use the format `{ VAR }` where `VAR` is the name of the varia
 - `repo` is the repository linked in `Cargo.toml`
 - `bin` is the name of a specific binary, inferred from the crate configuration
 - `target` is the rust target name (defaults to your architecture, but can be overridden using the `--target` command line option if required()
-- `archive-format` is the filename extension of the package archive format
+- `archive-suffix` is the filename extension of the package archive format, e.g. `.tar.tgz` for tgz or `.exe`/`""` for bin.
+- `archive-format` is the filename extension of the package archive format, e.g. `tar.tgz` for tgz or `exe`/`""` for bin.
```

Comment:
let's add in plain text that one includes the dot and the other not, and that suffix is preferred and format deprecated

Reply:
Done.